docs/site/Loopback-component-authentication.md

```diff
@@ -608,6 +608,23 @@ export class MyApplication extends BootMixin(
 }
 ```
 
+### Use passport based strategies
+
+The earlier version of `@loopback/authentication` is based on a express
+middleware called `passport`, which supports 500+ passport strategies for
+verifying express app's requests. In `@loopback/authentication@2.0`, we defined
+our own interface `AuthenticationStrategy` that describes a strategy with
+different contracts than the passport strategy, but we still want to keep the
+ability to support those existing 500+ community passport strategies. Therefore
+we rewrote the adapter class which convert a passport strategy to the one that
```

Comment:
> we rewrote the adapter class which convert a passport strategy to the one that

we rewrote the adapter class. It now converts a passport strategy to the one that